### PR TITLE
[Style] Add styles front-end

### PR DESCRIPTION
**Pull request by jesusMava**
Add styles to  views, they aren't perfect but I tried x.x

| Before| After |
| --------- | -------- |
|image|image|
|image|image|

### CODE REVIEW

File: app/views/curiosity_cards/_form.html.erb [hunk 1]
```diff
@@ -0,0 +1,49 @@
+<div class="columns is-mobile">
+  <div class="column is-half is-offset-one-quarter">
+    <%= form_with model: curiosity do |f| %>
+      <div class:'field'>
```
Comment (3zcurdia) [inline app/views/curiosity_cards/_form.html.erb:4]: html o ruby ?
```
<div class='field'>
```

File: app/views/curiosity_cards/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+<h2 class='title has-text-centered	'>New Curiosity</h2>
```
Comment (3zcurdia) [inline app/views/curiosity_cards/new.html.erb:1]: porque tantos espacios?

File: app/views/devise/registrations/new.html.haml [hunk 3]
```diff
@@ -1,4 +1,4 @@
-%h2 Sign up
+%h2.title Sign up
```
Comment (itsmeurbi) [inline app/views/devise/registrations/new.html.haml:1]: You also has a mixture between herb and haml.
Maybe not changing it in this PR, but remember, keep consistency is important

**Review by itsmeurbi — CHANGES_REQUESTED** [review]

File: app/views/curiosity_cards/_form.html.erb (hunk 1 above)
Comment (jesusMava) [reply app/views/curiosity_cards/_form.html.erb:4]: my bad, thanks

File: app/views/devise/registrations/new.html.haml (hunk 3 above)
Comment (jesusMava) [reply app/views/devise/registrations/new.html.haml:1]: Sure, actually I'll open a new PR to replace haml by erb

File: app/views/curiosity_cards/new.html.erb (hunk 2 above)
Comment (jesusMava) [reply app/views/curiosity_cards/new.html.erb]: my bad x2

**Review by itsmeurbi — APPROVED** [review]
LGTM

**Pull request merged**